app/controllers/jobs_controller.rb

```diff
@@ -2,8 +2,12 @@ class JobsController < ApplicationController
   before_action :authorize!
 
   def index
-    @jobs = Job.where.not(id: current_user.jobshidden.pluck(:id))
-    .includes(:company)
+    if current_user && params[:my_jobs]
+      @jobs = Job.find(current_user.likedjobs.pluck(:job_id))
+    else
+      @jobs = Job.where.not(id: current_user.jobshidden.pluck(:id))
+        .order(posting_date: :desc).includes(:company)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.